src/transactions/test/ManageBuyOfferTests.cpp

```diff
@@ -144,13 +144,60 @@ TEST_CASE("manage buy offer failure modes", "[tx][offers]")
 
         SECTION("check offer valid")
         {
-            SECTION("sell no issuer")
-            {
-                auto a1 = root.create("a1", minBalance1PlusFees);
-                REQUIRE_THROWS_AS(
-                    a1.manageBuyOffer(0, makeAsset(getAccount("fake"), "CUR3"),
-                                      native, Price{1, 1}, 1),
-                    ex_MANAGE_BUY_OFFER_SELL_NO_ISSUER);
+            SECTION("no issuer")
```

Comment:
Fixed

Reply:
I think you missed this one.

Reply:
This section is required because it holds code that removes the issuer, and I don't want this to affect the other tests. I believe the section was unnecessary in my initial commit, but then I moved common code into this section. I should've mentioned this in my comment instead of "Fixed".